README.md

```diff
@@ -114,6 +114,22 @@ Lego is not (and will never be):
 
 🎈 You can [read the core code](https://github.com/Polight/lego/blob/master/src/lib/Component.js) that is only 61 <abbr title="Lines Of Code">LOC</abbr> resulting that Lego is **as light as 3Kb** for the full bundle!
 
+## Developing and contributing on Lego
```

Comment:
I have added this section because at first I didn't know how to start developing on Lego. If you think about any relevant information to add, tell me :wink:

Reply:
suggestion: "Developing and Contributing to Lego"

Reply:
Done :heavy_check_mark: